huntsman/utils/hue_light_control.py

```diff
@@ -211,13 +211,28 @@ def hue_observing_mode(bridge, led_index, desk_index, verbose=False):
         bridge.lights[desk_index].state(on=False)
  
     
+   
+    
+    
+    
     
 class Boo:
     
     def hello(x, y):
```

Comment:
Should probably experiment with this stuff in another branch or somewhere else. ;)

Also, the first param to all methods inside a class should be `self`